src/pages/python/lambda-expressions/index.md

```diff
@@ -3,10 +3,19 @@ title: Lambda Expressions
 ---
 ## Lambda Expressions
 
-Lambda Expressions are ideally used when we have something simple to be done an we are more interested in quickly getting the job done rather than formally naming the function.
-<a href='https://github.com/freecodecamp/guides/tree/master/src/pages/python/lambda-expressions/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+Lambda expression is a breakdown of a function.
```

Comment:
I'm not sure what you mean by "a breakdown" of a function. Can you explain what you mean?

Reply:
What I mean by that is that lambda expression acts as like an anonymous function. When transforming regular functions into lambda expressions, we simplify the function (deleting some keywords from the function or breaking down the function) by deleting the keyword def and the return statement.

Maybe I should change it to "Lambda expression is an anonymous function"

Reply:
That seems to be clearer, but even that assumes you know what an anonymous function is.

Reply:
How about this "Lambda expression is an anonymous function - a function that doesn't have a name"